R/conversion.R

```diff
@@ -91,12 +91,9 @@ convert <- function(text,
 
   # Define Pattern for latex code
 
-  unicode_latex$int <- as.integer(as.hexmode(unicode_latex$unicode))
-  char_latex <- ifelse(unicode_latex$int <= 255 & unicode_latex$int != 177, unicode_latex$chr,
-    ifelse(unicode_latex$int < 32768,
-      paste0("\\uc1\\u", unicode_latex$int, "*"),
-      paste0("\\uc1\\u", unicode_latex$int - 65536, "*")
-    )
+  unicode_int <- as.integer(as.hexmode(unicode_latex$unicode))
```

Comment:
Assigning to `unicode_latex$int` means that R will have to make a new copy of `unicode_latex` here (i.e., copy-on-write), which is unnecessary. Since `unicode_latex$int` is not used outside this function, I'd just use a normal variable here, e.g., `unicode_init`.

BTW, where is `unicode_latex` from? I see there is a `sysdata.rda` but I don't know how it was created. You can probably pre-compute and store the integer code for the Unicode characters in `unicode_latex`, so you won't need to do the integer conversion here.

Reply:
Thanks for checking the source. It was downloaded from `https://milde.users.sourceforge.net/LUCR/Math/data/unimathsymbols.txt`

Originally we documented in the `data.R` but removed after we save the data into `sysdaata.rds` per #50. 

https://github.com/Merck/r2rtf/commit/021f96cebc535a05d67fdce8ed0294ade5de317c

Reply:
I see. In that case, you may consider writing the data directly into an R script, like what I did for in the **mime** package: the MIME types https://github.com/yihui/mime/blob/main/R/mimemap.R are dynamically generated from https://github.com/yihui/mime/blob/1763e0dcb72fb58d97bab97bb834fc71f1e012bc/R/mime.R#L20-L55

I think that will make the source code more transparent, and you wouldn't need the `globalVariables()` hack in `zzz.R` (personally I'd avoid using `globalVariables()` unless absolutely necessary).

Reply:
I created an issue for this: https://github.com/Merck/r2rtf/issues/218